### PR TITLE
Add EQ effect for AudioSegment

### PR DESCRIPTION
**Pull request by anubhav-narayan**

### CODE REVIEW

**Comment by jiaaro** [comment]
thanks, this is a useful and long-overdue feature.

Would you mind removing all the extraneous `pass` statements and adding spaces around the assignments (e.g., `x = 1` instead of `x=1` )?

File: pydub/scipy_effects.py [hunk 1]
```diff
@@ -62,3 +62,109 @@ def high_pass_filter(seg, cutoff_freq, order=5):
...
+		return ms_to_stereo(seg)
+		pass
+	if channel_mode=="M":
+		seg=stereo_to_ms(seg)
+		seg=[_eq(seg[0],focus_freq,bandwidth,filter_mode,gain_dB,order),seg[1]]
+		seg=AudioSegment.from_mono_audio_segements(seg[0],seg[1])
+		return ms_to_stereo(seg)
+		pass
+	if channel_mode=="S":
+		seg=stereo_to_ms(seg)
```
Comment (jiaaro) [inline pydub/scipy_effects.py:165]: This code seems to assume it's getting back a list of mono audio segments. but stereo_to_ms returns a single stereo AudioSegment. code like `seg[0]` is going to operate on just the first 1ms of audio

Comment (anubhav-narayan) [reply pydub/scipy_effects.py]: Found that bug. Fixed it.

**Comment by anubhav-narayan** [comment]
I was thinking if I could create a module/library which has tools exclusively for musical applications using pydub and since I have no experience of making a python library could you help me with that?

**Comment by jiaaro** [comment]
is #463 included in this PR?

**Comment by anubhav-narayan** [comment]
Yes it does.

**Pull request merged**